### PR TITLE
DOC: Add docstring examples for stats.binom_test

### PR DESCRIPTION
**Pull request by camilog04**
DOC: Add docstring example for stats.binom_test

### CODE REVIEW

File: scipy/stats/morestats.py [hunk 1]
```diff
@@ -2035,6 +2035,20 @@ def binom_test(x, n=None, p=0.5, alternative='two-sided'):
...
 
+    Examples
+    --------
+    >>> from scipy import stats
+
+    A car manufacturer claims that no more than 10% of their cars are unsafe. 
+    15 cars are inspected for safety, 3 were found to be unsafe. Test the 
+    manufacturer's claim:
+
+    >>> stats.binom_test(3, n=15, p=0.1, alternative = 'greater')
```
Comment (chrisb83) [inline scipy/stats/morestats.py]: Remove spaces around =

File: scipy/stats/morestats.py [hunk 2]
```diff
@@ -2035,6 +2035,20 @@ def binom_test(x, n=None, p=0.5, alternative='two-sided'):
...
+    0.18406106910639114
+
+    The null hypothesis cannot be rejected at the 5% level of significance
+    because the returned p-value is greater than the creitical value of 5%.
```
Comment (chrisb83) [inline scipy/stats/morestats.py]: Typo: critical

**Review by chrisb83 — COMMENTED** [review]
Thanks for the PR, looks good except minor details

File: scipy/stats/morestats.py (hunk 1 above)
Comment (kalash04) [reply scipy/stats/morestats.py]: done

**Comment by kalash04** [comment]
@chrisb83  thank you for the feedback I corrected it.

**Review by chrisb83 — APPROVED** [review]

**Comment by WarrenWeckesser** [comment]
Thanks @kalash04!

**Pull request merged**